rootfs/scheduler/tests/test_deployments.py

```diff
@@ -73,6 +76,41 @@ def scale(self, namespace=None, name=generate_random_name(), **kwargs):
         self.scheduler.scale(namespace, name, **kwargs)
         return name
 
+    def test_deployment_api_version_1_9_and_up(self):
```

Comment:
This is very nice test! Can you add a couple of tests for version number `1.10+` (double digits after the period) and for example for GKE has version numbers `1.12.9-gke.3`. We need to also make the parser work for conversioning like this.

It would be nice if the test already covered some of those scenarios too! :)

Reply:
Still getting the following exception on `v1.13.7-gke.8`: 

```
Traceback (most recent call last):
  File "/app/api/models/appsettings.py", line 181, in save
    self.update_autoscale(previous_settings)
  File "/app/api/models/appsettings.py", line 120, in update_autoscale
    self.app.autoscale(proc, scale)
  File "/app/api/models/app.py", line 975, in autoscale
    self._scheduler.hpa.get(self.id, name)
  File "/app/scheduler/resources/horizontalpodautoscaler.py", line 35, in get
    url = self.api(url, *args)
  File "/app/scheduler/resources/__resource.py", line 28, in api
    return "/{}/{}".format(self.api_prefix, self.api_version) + tmpl.format(*args)
  File "/app/scheduler/resources/horizontalpodautoscaler.py", line 16, in api_version
    if self.version() >= parse("1.3.0"):
  File "/app/scheduler/__init__.py", line 87, in version
    return Version('{}.{}'.format(data['major'], data['minor']))
  File "/usr/local/lib/python3.5/dist-packages/packaging/version.py", line 202, in __init__
    raise InvalidVersion("Invalid version: '{0}'".format(version))
packaging.version.InvalidVersion: Invalid version: '1.13+'
```

Reply:
@Cryptophobia absolutely, bud! I'm out of commission this week, but will take care of this as soon as I'm back at a computerbox after this coming weekend.

Thanks for the heads up on the GKE version reporting. Weirdly, I think that *might* be unrelated to the changes already made, but are likely related to changes that I've yet to make (discovered in testing with @kingdonb).

To that end, absolutely any information you can provide about the failure scenario you hit would be very much appreciated. :)